EIPS/eip-1.md

```diff
@@ -37,11 +37,31 @@ An EIP must meet certain minimum criteria. It must be a clear and complete descr
 
 ## EIP Work Flow
 
+### Shepherding an EIP
+
 Parties involved in the process are you, the champion or *EIP author*, the [*EIP editors*](#eip-editors), and the [*Ethereum Core Developers*](https://github.com/ethereum/pm).
 
-:warning: Before you begin, vet your idea, this will save you time. Ask the Ethereum community first if an idea is original to avoid wasting time on something that will be be rejected based on prior research (searching the Internet does not always do the trick). It also helps to make sure the idea is applicable to the entire community and not just the author. Just because an idea sounds good to the author does not mean it will work for most people in most areas where Ethereum is used. Examples of appropriate public forums to gauge interest around your EIP include [the Ethereum subreddit], [the Issues section of this repository], and [one of the Ethereum Gitter chat rooms]. In particular, [the Issues section of this repository] is an excellent place to discuss your proposal with the community and start creating more formalized language around your EIP.
+It will be your role as the champion to socialize your EIP and build community consensus around it. Before you begin writting a formal EIP, you should vet your idea. Ask the Ethereum community first if an idea is original to avoid wasting time on something that will be be rejected based on prior research. It is recommended to open a discussion thread on [the Ethereum Magicians forum], but you can also use [one of the Ethereum Gitter chat rooms], [the Ethereum subreddit] or [the Issues section of this repository]. 
```

Comment:
```suggestion
It will be your role as the champion to socialize your EIP and build community consensus around it. Before you begin writing a formal EIP, you should vet your idea. Ask the Ethereum community first if an idea is original to avoid wasting time on something that will be be rejected based on prior research. It is recommended to open a discussion thread on [the Ethereum Magicians forum], but you can also use [one of the Ethereum Gitter chat rooms], [the Ethereum subreddit] or [the Issues section of this repository]. 
```

Reply:
@fubuloubu maybe it's because I haven't had my coffee yet, but the two lines look identical to me. What exactly did you change? Thanks!

Reply:
Typo: `writting` -> `writing`

Did GitHub not highlight?

Reply:
Good catch, thanks! No, it didn't highlight :-/ 

<img width="108" alt="Screen Shot 2019-05-07 at 4 04 38 PM" src="https://user-images.githubusercontent.com/9390255/57329466-d3825f80-70e1-11e9-86a5-7122a18ba245.png">